tox.ini

```diff
@@ -35,6 +35,8 @@ commands =
 [flake8]
 ignore = E712, E711, C901
 max-complexity = 10
+max-line-length = 80
+exclude = flaskbb/configs/default.py,flaskbb/_compat.py
```

Comment:
Flake8 complains a lot about the lambda assignments in `_compat` and `default` is a settings file that Flake8 doesn't like. It might be worth it to fix its complaints in these files but I'm not sure.

Line-length 80 because there were a number of lines that were 80 chars long with no good way to fix without making a mess of it.